File: Practice/java/LeapYear.java
import java.util.Scanner;

public class LeapYear {
  public static void main (String[] args) {
    System.out.println("西暦を入力してください");

    Scanner scan = new Scanner(System.in);

    int year = scan.nextInt();

    if (year%4 == 0 && year%100 != 0 || year%400 == 0) {
      System.out.println(year + "年は閏年です");
    } else {
      System.out.println(year + "年は閏年ではありません");
    }
  }
}

File: Practice/cpp/LeapYear.cpp
#include <iostream>
using namespace std;

int main() {
  int year;

  cout << "西暦を入力してください: ";
  cin >> year;

  if (year%4 == 0 && year%100 != 0 || year%400 == 0) {
    cout << year << "年は閏年です" << endl;
  } else {
    cout << year << "年は閏年ではありません" << endl;
  }
}
